admins/README.md

```diff
@@ -0,0 +1,129 @@
+# Administrators
+
+## Table of Contents
```

Comment:
Per our [`markdowdlint` rules](https://github.com/BonnyCI/lore/blob/master/.markdownlint.js), please add a newline after this header:
```markdown
## Table of Contents

- [Components](#components)
```
This is also the case for each of the sections below.